src/test/java/pages/AccountModel.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Random;
+
+public class AccountModel {
```

Comment:
This class is not model, this class is page object, that's why AcoountPage or something like that. "Model" classes just store information